model/src/game_mechanics/pets.rs

```diff
@@ -0,0 +1,40 @@
+use serde::{Deserialize, Serialize};
+
+use crate::*;
+pub use crate::game_mechanics::skills;
```

Comment:
`SkillId` seems to be the only used import from `skills`, please import it directly then.